libminifi/src/sitetosite/RawSocketProtocol.cpp

```diff
@@ -112,11 +112,11 @@ bool RawSiteToSiteClient::initiateResourceNegotiation() {
     return false;
   }
 
-  logger_->log_info("Negotiate protocol version with destination port %s current version %d", port_id_str_, _currentVersion);
+  logger_->log_debug("Negotiate protocol version with destination port %s current version %d", port_id_str_, _currentVersion);
 
   int ret = peer_->writeUTF(getResourceName());
 
-  logger_->log_info("result of writing resource name is %i", ret);
+  logger_->log_debug("result of writing resource name is %i", ret);
```

Comment:
^ this honestly seems like it would be a trace level to help trace the level of execution through the stream writes.